topostats/validation.py

```diff
@@ -25,10 +25,31 @@ def validate_config(config: dict):
             "cores": lambda n: 1 <= n <= os.cpu_count(),
             "quiet": Or(True, False, error="Invalid value in config for 'quiet', valid values are 'True' or 'False'"),
             "file_ext": Or(
-                ".spm", ".jpk", error="Invalid value in config for 'file_ext', valid values are '.spm' or '.jpk'"
+                ".spm",
+                ".jpk",
+                ".ibw",
+                error="Invalid value in config for 'file_ext', valid values are '.spm', '.jpk' or '.ibw'",
             ),
             "loading": {
-                "channel": Or("Height"),
+                "channel": Or(
+                    "ZSensor",
+                    "",
```

Comment:
Are there meant to be channels without names?

Reply:
Just to put here what we spoke about:
This can be misleading but it's present nonetheless. I added the below comment incase we decide that it would also be useful to print/log the channel description list which could make sense of it (line 112 in io.py):
```# channel_description = channel.decode('latin1').split('"')[1] # incase the blank field raises quesions?```
Although, a description is not present for all filetypes it seems